sphinx/source/recovery.rst

```diff
@@ -22,9 +22,8 @@ To initiate the first phase of the recovery protocol, one of several nodes must
 
 .. code-block:: bash
 
-    $ cchost --start=recover --enclave-file=/path/to/application --raft-host=raft_ip --raft-port=raft_port
-    --tls-host=tls_ip --tls-pubhost=tls_public_ip --tls-port=tls_port --ledger-file=ledger_file
-    --node-cert-file=/path/to/node_certificate --quote-file=/path/to/quote
+    $ cchost --start=recover --enclave-file=/path/to/application --node-address=node_ip:node_port --rpc-address=tls_ip:tls_port
+    --ledger-file=ledger_file --node-cert-file=/path/to/node_certificate --quote-file=/path/to/quote
```

Comment:
Interestingly, I think we will be able to get rid of those last two arguments in the new light genesis startup mode.